config/settings.yml

```diff
@@ -37,3 +37,9 @@ preservation_catalog:
 email_addresses:
   discussion_list: 'user@discussion_list.org'
   user_list: 'user@user_list.org'
+
+# we expect/hope that this will not stay around forever.  see usage for further explanation.
+hacks:
+  update_catalog_sleep:
+    base_time: 0.05    # the default values are artificially low to keep test suite fast -- should override to be many seconds in prod
```

Comment:
are we sure sleep is in seconds, not milliseconds?

Reply:
Docs say "Suspends the current thread for duration seconds (which may be any number, including a Float with fractional seconds)." https://apidock.com/ruby/v2_6_3/Kernel/sleep